### DISCUSSION TITLE
Option to show markdown code of other users comments

### DISCUSSION THREAD

**Discussion by tilkinsc** [community/community#31913] [Projects and Issues]
Whenever I go by a comment in an issue with some snazzy feature my immediate reaction is 'oh how did they do that!'

I would like the ... menu to have an option to see other's issue comments raw markdown.

* Users will get more versed in Github's features
* Ability to correct why others markdown code isn't working

**Comment by svick** [comment]
I agree that this would be nice. In the meantime, if you're willing to spend some effort, you can get access to the Markdown using the GitHub API. For issues and comments on them, you can use the REST API by going to an URL like https://api.github.com/repos/ipatix/agbplay/issues/51 and looking at the `body` element in the JSON.

For discussions, you're going to have to use the GraphQL API. For example:

```graphql
{
  repository(owner: "community", name: "community") {
    discussion(number: 31913) {
      body
    }
  }
}
```

You can see the result in [the GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer).

**Reply by tilkinsc** [reply]
Yo I actually forked that project recently 😂

**Comment by DenverCoder1** [comment]
One option is to "quote reply" to the message.

This will show the original markdown of the message, albeit prefixed with `>`.

If you remove the `>` marks (you can highlight the text and click the quote button), you will be able to replicate the message.

![image](https://user-images.githubusercontent.com/20955511/188317914-c42a7e62-d9bf-4fed-8b26-56c7fe27273a.png)

![image](https://user-images.githubusercontent.com/20955511/188317961-28b86ec6-0d73-42f5-874f-f7bc36fe47be.png)

**Reply by tilkinsc** [reply]
This is big brain thanks for work around!
